workbench-cypress/cypress/e2e/pulls/index.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { shouldBeVisible } from "@sharedTests/sharedFunctionsAndVariables";
+import { testDBHeader } from "@utils/sharedTests/dbHeaders";
+import { runTests } from "@utils/index";
+import { selectBranch } from "@utils/sharedTests/changeBranch";
+
+const pageName = "Pull requests page";
```

Comment:
This would represent a merged pull right? Page and file name should reflect that